src/lib/assets/scripts.js

```diff
@@ -0,0 +1,7 @@
+
+function show(shown, hidden) {
```

Comment:
- I think it's cleaner to use 2 functions for this (show/hide) or rename it to swap or something
- the console log must be removed
- instead of setting display block, `initial` is probably a better value